index.html

```diff
@@ -829,6 +831,70 @@ <h2><code>application/vc+ld+sd-jwt</code></h2>
       </table>
     </section>
   </section>
+  
+
+  <section id="json-web-token-claim-registrations">
+    <h3>JSON Web Token Claims</h3>
+    <p>
+      This section retained deprecated registered claim names, 
+      that have been submitted to the Internet Engineering Steering Group
+      (IESG) for review, approval, and registration with IANA in the
+      "JSON Web Token Claims Registry".
```

Comment:
Fine, save the link addition for later. But these text tweaks (relocated from above) are focused on your "Add deprecated claim names" changes.
```suggestion
      This section retains deprecated registered claim names 
      that were previously submitted to the Internet Engineering Steering
      Group (IESG) for review, approval, and registration with IANA in the
      "JSON Web Token Claims Registry".
```

Reply:
I'm good to merge these as is when https://github.com/w3c/vc-data-model/pull/1195 is merged

Reply:
I guess I can take them now if there are no objections.